src/main/webapp/projects/index.jsp

```diff
@@ -69,7 +69,7 @@
                 <div class="ignite_project_type">Client</div>
                     <div class="ignite_project_content">
                         <a href="pade/index.jsp" class="ignite_project_pade">
-                        <h1>Pàdé <span><%= Versions.getVersion("pade") %></span></h1></a>
```

Comment:
This removes whitespace - is that intentional?

Reply:
Nope. Oversight. Forgot it had a purpose. Probably should use &nbsp; instead. Thanks

Reply:
I'm not sure if it has a purpose, to be honest. Better safe than sorry. :)